scylla-server-rust/src/services/data_service.rs

```diff
@@ -0,0 +1,49 @@
+use prisma_client_rust::{chrono::DateTime, QueryError};
+
+use crate::{
+    prisma::{self},
+    serverdata::ServerData,
+    Database,
+};
+
+/// Get a datapoint
+pub async fn get_data(
+    db: &Database,
+    data_type_name: String,
+    run_id: i32,
+) -> Result<Vec<prisma::data::Data>, QueryError> {
+    db.data()
+        .find_many(vec![
+            prisma::data::data_type_name::equals(data_type_name),
+            prisma::data::run_id::equals(run_id),
+        ])
+        .exec()
+        .await
+}
+
+/// Add a datapoint
```

Comment:
All the service methods should have robust comments specifying parameter types return types, and any weird logic. We use JavaDoc style for FinishLine, looks like RustDoc is the equivalent for Rust

Reply:
Example of a FinishLine one:
  /**
   * Retrieves a single design review
   *
   * @param submitter the user who is trying to retrieve the design review
   * @param designReviewId the id of the design review to retrieve
   * @param organizationId the organization that the user is currently in
   * @returns the design review
   */

Reply:
rustdoc doesnt support any annotations such as params, so it wont parse them or anything nice :( Added in latest commit.